static/src/js/components/AccessMethod/Select.scss

```diff
@@ -0,0 +1,89 @@
+$trigger-width: 400px;
+
+.SelectTrigger {
+  width: $trigger-width;
+  border-radius: 4px;
+  border: 1px solid $color__black--200;
+  padding: 0 15px;
+  font-size: 13px;
```

Comment:
We should be using `rem`s for all fixed sizing. That includes text, margins, padding, etc. We rely on that behavior to do some nice scaling depending on screen size. You can get the proper size in rems by dividing the px value by 16. For example `13px / 16 = 0.8125rem`. For margins and padding, I like to calculate the variable based of the `$spacer` var (when I can remember 😅) so thats all consistent.